docs/en/concepts-and-designs/README.md

```diff
@@ -23,3 +23,5 @@ SkyWalking already support.
   - [Overview](ui-overview.md). A simple brief about SkyWalking UI.
 - CLI
   - [SkyWalking CLI](https://github.com/apache/skywalking-cli). A command line interface for SkyWalking.
+- Events
+  - [Events](event.md). Introduce the concept of "Event" in SkyWalking.
```

Comment:
I think this should be listed here? https://github.com/apache/skywalking/blob/845312f28b61bf51beba5a3205dd04e38c0ae08b/docs/en/concepts-and-designs/overview.md#why-use-skywalking

Reply:
I'll list it there once the doc is settled down

Reply:
Some other documents should be updated
1. Receiver documentation
2. [Protocol doc](https://github.com/apache/skywalking/blob/master/docs/en/protocols/README.md), should add whole new section in `Probe Protocols`, as `Event Report Protocol` before `3rd-party instrument protocol`